### ISSUE TITLE
Support for subdomains

### ISSUE THREAD

**Issue by justinperkins** [basecamp/mission_control-web#43]
Apologies for jumping on the feature request bandwagon out of the gate, but hear me out ...

tl;dr: Optional support for blocking a path+subdomain would be really nice

I'd like to add this to a production multi-tenant app, and I know that historically cascading performance issues typically stem from latency spikes arising with certain tenants at certain path requests. I have often wished I had a toggle at my fingertips where I could disable path X for customer Y, as opposed to just path X across the board.

Disabling path X for all customers is not only overkill, but ultimately not something I can do very easily on the app I'm working on without a lot of red tape and team communications, approvals, and advanced customer notifications. 

But disabling path X for customer Y would be no problem (from an operations perspective) and also more often than not, perfectly address the application performance urgency so that the team has the breathing room to deescalate the emergency.
